src/main/scala/nl.knaw.dans.easy.curationwork/Command.scala

```diff
@@ -28,22 +27,35 @@ import scala.util.{ Failure, Success, Try }
 object Command extends App with DebugEnhancedLogging {
   type FeedBackMessage = String
 
-  val configuration = Configuration(Paths.get(System.getProperty("app.home")))
   val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
     verify()
   }
-  val app = new EasyManageCurationWorkApp(configuration)
+  val reporter = new Report(configuration)
+  val assigner = new Assign(configuration)
+  val unassigner = new Unassign(configuration)
 
-  runSubcommand(app)
-    .doIfSuccess(msg => println(s"OK: $msg"))
+  runSubcommand(reporter, assigner, unassigner)
+    .doIfSuccess(msg => println(s"$msg"))
     .doIfFailure { case e => logger.error(e.getMessage, e) }
     .doIfFailure { case NonFatal(e) => println(s"FAILED: ${ e.getMessage }") }
 
-  private def runSubcommand(app: EasyManageCurationWorkApp): Try[FeedBackMessage] = {
+  private def runSubcommand(reporter: Report, assigner: Assign, unassigner: Unassign): Try[FeedBackMessage] = {
```

Comment:
since these values are already 'globally' available (relative to this function, anyway), why do you need to pass them along as parameters?

Reply:
Correct. I  had simply replaced 'app' with the three new instances. It is indeed not needed to give these as parameters.